src/audio/asrc/asrc.c

```diff
@@ -0,0 +1,616 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2019 Intel Corporation. All rights reserved.
```

Comment:
2012-2019

Reply:
This is a brand new file developed for SOF so I used 2019.